tasmota/xdrv_62_robotdyn.ino

```diff
@@ -0,0 +1,253 @@
+/*
+  xdrv_62_robotdyn.ino -support for robotdin 4 AC channel dimmer with zero crossing
+
+  SPDX-FileCopyrightText: 2022 Benoit Lecuppe
+
+  SPDX-License-Identifier: GPL-3.0-only
+*/
+//ifdef VSCODE_DEV
+  #define USE_LIGHT
+  #define USE_ROBOTDYN_DIMMER
+//endif
+#ifdef USE_LIGHT
+#ifdef USE_ROBOTDYN_DIMMER
+/*********************************************************************************************\
+ * RobotDyn AC Power dimmer controller WI-FI 
+ * ZeroCross GPIO5
+ * DIMMER1   GPIO4
+ * DIMMER2  GPIO13
+ * DIMMER3  GPIO14
+ * DIMMER4  GPIO12
+ * MCU ESP8266
+ * FLASH 4MB
+ * AC-IN 90-240V
+ * DC - VDD 3.3V
+ * RDB abreviation mean RoBotDyn
+\*********************************************************************************************/
+
+#define XDRV_62                 62
+
+extern "C" int startWaveform(uint8_t pin, uint32_t timeHighUS, uint32_t timeLowUS, uint32_t runTimeUS = 0, int8_t alignPhase = -1, uint32_t phaseOffsetUS = 0, bool autoPwm = false);
+extern "C" int stopWaveform(uint8_t pin);
+
+
+void rbdInit(void){
+  // RobotDyn AC dimmer Init
+  AddLog(LOG_LEVEL_INFO, PSTR("========== RBD INIT rdbInit() "));
+  // init ZeroCross detection. One pin must be attach to "ZC Pulse"
+  if (!PinUsed(GPIO_ZEROCROSS))  AddLog(LOG_LEVEL_ERROR, PSTR("========== RBD INIT ERROR: on pin mist be set to ZC Pulse in template"));
```

Comment:
Change all AddLog(LOG_LEVEL_x, PSTR("====) functions to AddLog(LOG_LEVEL_x, PSTR("RBD: ) in line with Tasmota logging.

Also use text strings as short as possible, You're not writing a book! It should be cheatsheet.